docs/spellcheck_exceptions.txt

```diff
@@ -0,0 +1,98 @@
+APIs
+BashActivator
+CPython
+CShellActivator
+DSL
+Dex
+EmptyDirVolumeSource
+FishActivator
+FromAppData
+GCP
+Github
+IAP
```

Comment:
Maybe it is possible to do case-insensitive string comparison so not put the same entry multiple times?

Reply:
yes, found an option and changed that